plone/app/contenttypes/tests/test_webdav.py

```diff
@@ -0,0 +1,56 @@
+import os.path
+from plone.app.contenttypes.testing import (
+    PLONE_APP_CONTENTTYPES_INTEGRATION_TESTING,
+)
+from plone.app.testing import TEST_USER_ID, setRoles
+import unittest2 as unittest
+from zope.publisher.browser import TestRequest
+from ZPublisher.HTTPResponse import HTTPResponse
+
+
+class DAVTestRequest(TestRequest):
+
+    get_header = TestRequest.getHeader
+
+    def _createResponse(self):
+        return HTTPResponse()
+
+
+class ImageViewIntegrationTest(unittest.TestCase):
+
+    layer = PLONE_APP_CONTENTTYPES_INTEGRATION_TESTING
+
+    def setUp(self):
+        self.portal = self.layer['portal']
+        self.request = self.layer['request']
+        setRoles(self.portal, TEST_USER_ID, ['Manager'])
+        self.portal.invokeFactory('Image', 'image')
+        self.image = self.portal['image']
+        self.image.title = "My Image"
+        self.portal.invokeFactory('File', 'file')
+        self.file = self.portal['file']
+        self.file.title = "My file"
+
+    def test_image_put(self):
```

Comment:
I prefer docstrings in tests, describing what it does. But this is only personal preference and might be obvious in this case anyways.